### PR TITLE
Dockerize Takwimu

### LINKED ISSUE #21: Dockerize TAKWIMU
## Description
To improve development and deployment, we should add a Docker workflow.

## Tasks

- [x] Add Dockerfile
- [ ] Create DockerHub organisation + repo
- [x] Add docker-compose.yml
- [ ] Create Make file similar to https://github.com/CodeForAfrica/openAFRICA

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

To improve development and deployment, we should add a Docker workflow.
Fixes #21 

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+COPY requirements.txt /app/
+
+RUN pip install -r /app/requirements.txt
+
+ADD . /app/
+
+# Django service
+EXPOSE 8000
+
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]
```
Comment (DavidLemayian) [inline Dockerfile]: Run with gunicorn instead?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ENV PYTHONUNBUFFERED 1
+ENV DATABASE_URL postgresql://takwimu:takwimu@db:5432/takwimu
+
+RUN mkdir /app
+WORKDIR /app
+
+# Installing OS Dependencies
+RUN apt-get update && apt-get upgrade -y
+
+RUN pip install -U pip setuptools
```
Comment (DavidLemayian) [inline Dockerfile]: Upgrade pip with this too.

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+version: '3.3'
+
+services:
+ # Postgres
+ db:
```
Comment (DavidLemayian) [inline docker-compose.yml]: Is the indentation here correct?

File: Dockerfile (hunk 2 above)
Comment (phillipahereza) [reply Dockerfile]: @DavidLemayian pip is being uploaded here

Comment (DavidLemayian) [reply Dockerfile]: @phillipahereza Missed the -U flag. thanks.

**Pull request merged**